chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2799,4 +2802,36 @@ public int getYClipMax() {
   public void setYClipMax(int yClipMax) {
     this.yClipMax = yClipMax;
   }
+
+  /**
+   * This will get (and resolve) the scene specific folder in the scenes directory.
```

Comment:
*and create it if it doesn't exist

Maybe rename this method to `ensureSceneDirectory` and put it into `RenderContext`? This method is used for a scene but isn't really related to a scene's content.